src/ducks/Watchlists/gql/list/mutations.js

```diff
@@ -47,12 +47,12 @@ export function useUpdateWatchlist (type) {
         ? watchlist.isMonitored
         : newParams.isMonitored
 
-    const listItems =
-      type === 'SCREENER'
-        ? []
-        : newParams.listItems
-        ? newParams.listItems
-        : watchlist.listItems
+    const listItems = (type === 'SCREENER'
+      ? []
```

Comment:
to be honest. When I see ? : ? : chain, I really want to see a normal if else clause.